src/iota/buffer.rs

```diff
@@ -211,6 +240,41 @@ impl Buffer {
 
 }
 
+/// If c1 -> c2 is a word edge
+fn is_word_edge(c1: &u8, c2: &u8, whitespace_only: bool) -> bool {
+    const WHITESPACE: &'static str = " \n\t";
+    const WORD_CHARS: &'static str = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_";
```

Comment:
If you cast c1 and c2 to `char`s you could use the `is_alphabetic` method, which would mean you don't need these consts, and would clean up this function quite a bit.